Content.Client/PDA/PdaMenu.xaml.cs

```diff
@@ -161,7 +167,9 @@ public void UpdateState(PdaUpdateState state)
             _stationName = state.StationName ?? Loc.GetString("comp-pda-ui-unknown");
             StationNameLabel.SetMarkup(Loc.GetString("comp-pda-ui-station",
                 ("station", _stationName)));
-            
+
+            _balance = Loc.GetString("comp-pda-ui-balance", ("balance", state.Balance)); // Frontier
+            BalanceLabel.SetMarkup(_balance); // Frontier
```

Comment:
This doesn't work so well with the OnPressed copy ability (I didn't know you could even do that!) - it'll copy the whole markdown string.  Recommending this, works better locally, more consistent with other entries:
```suggestion
            _balance = state.Balance.ToString(); // Frontier
            BalanceLabel.SetMarkup(Loc.GetString("comp-pda-ui-balance", ("balance", _balance))); // Frontier
```